vanilla-installer/gui.py

```diff
@@ -1,13 +1,23 @@
 """Runs the GUI for VanillaInstaller."""
 # IMPORTS
-import os
-import sys
-import tkinter
-import tkinter.filedialog
 import argparse
 import webbrowser
 import pathlib
-import logging
+from PySide6.QtCore import QCoreApplication, QRect, Qt, QRunnable, QThreadPool, Slot
+from PySide6.QtGui import QFont
+from PySide6.QtWidgets import (
+    QApplication,
+    QComboBox,
+    QLabel,
+    QMainWindow,
+    QPushButton,
+    QTextEdit,
+    QWidget,
+    QGraphicsColorizeEffect,
+    QFileDialog
+)
+from PySide6.QtSvgWidgets import QSvgWidget
+import minecraft_launcher_lib as mll
 
 # External
 import requests
```

Comment:
```suggestion

```
unused import, not a huge deal though
(same with logger but I can't leave a review comment there)